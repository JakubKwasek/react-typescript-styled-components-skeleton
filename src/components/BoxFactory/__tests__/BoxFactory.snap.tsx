import * as React from "react";
import * as renderer from "react-test-renderer";
import { BoxFactory } from "../";

describe("BoxFactory", () => {
	it("renders correctly", () => {
		const tree = renderer
			.create(<BoxFactory width={"xsmall"}><p>Test content</p></BoxFactory>)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
