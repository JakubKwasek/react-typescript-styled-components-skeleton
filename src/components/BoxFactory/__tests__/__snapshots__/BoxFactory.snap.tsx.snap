// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoxFactory renders correctly 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 hlxvTc"
>
  <p>
    Test content
  </p>
</div>
`;
